src/Resources/views/DashboardAdmin/compose_container_show.html.twig

```diff
@@ -18,7 +18,10 @@
     <div class="dashboard-composer__block-type-selector">
         <label>{{ 'composer.block.add.type'|trans({}, 'SonataDashboardBundle') }}</label>
         <select class="dashboard-composer__block-type-selector__select" style="width: auto">
-            {% for blockServiceId, blockService in blockServices %}
+            {% for blockServiceId, blockService in blockServices|filter(service => service.metadata is defined) %}
```

Comment:
The `filter` filter requires `twig/twig:^1.41|^2.10`: https://twig.symfony.com/doc/2.x/filters/filter.html#filter
https://github.com/sonata-project/SonataDashboardBundle/blob/72fd3a1d1825676a1bf8d4c079aabf9b04129690/composer.json#L38

Reply:
Fixed